### PR TITLE
smsc95xx: Forward MII interrupts to phylib to avoid polling [REVPI-1886]

### PR DESCRIPTION
**Pull request by l1k**

### CODE REVIEW

File: drivers/net/usb/smsc95xx.c [hunk 1]
```diff
@@ -1154,6 +1154,14 @@ static const struct net_device_ops smsc95xx_netdev_ops = {
 	.ndo_set_features	= smsc95xx_set_features,
```
Comment (iluminat23) [inline drivers/net/usb/smsc95xx.c:1170]: This looks like an upstream patch, could you ad the upstream commit id?
 net: usb: Correct PHY handling of smsc95xx 
commit a049a30fc27c1cb2e12889bbdbd463dbf750103a upstream.

File: drivers/net/usb/smsc95xx.c [hunk 2]
```diff
@@ -2068,7 +2068,8 @@ static const struct driver_info smsc95xx_info = {
 	.bind		= smsc95xx_bind,
```
Comment (iluminat23) [inline drivers/net/usb/smsc95xx.c:2048]: net: usb: Correct reset handling of smsc95xx 
commit 0bf3885324a8599e3af4c7379b8d4f621c9bbffa upstream.

**Comment by nbuchwitz** [comment]
Packages for review: https://github.com/RevolutionPi/linux/actions/runs/2002965830

**Review by linosanfilippo-kunbus — DISMISSED** [review]

**Review by iluminat23 — CHANGES_REQUESTED** [review]
Please add the upstream commit ids.

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**